src/Eagle.ts

```diff
@@ -2475,13 +2490,71 @@ export class Eagle {
             }
         }
     }
+    
+    //dragdrop WIP
+
+    nodeDragStart = (eagle : Eagle, e : JQueryEventObject) => {
+        Eagle.nodeDropped = e.target;
+        $(".leftWindow").addClass("noDropTarget");
+        $(".rightWindow").addClass("noDropTarget");
+        $(".navbar").addClass("noDropTarget");
+
+        var drag = Eagle.nodeDropped.getElementsByClassName('input-group-prepend')[0] as HTMLElement;
```

Comment:
Maybe an extra comment here. I guess this is grabbing the icon used in the palette-component for use as the drag image, but it is a little dense.